snippets/csharp/VS_Snippets_CLR/HowToVerifyXMLDocumentRSA/cs/sample.cs

```diff
@@ -59,24 +59,24 @@ public static void Main(String[] args)
 
     // Verify the signature of an XML file against an asymmetric 
     // algorithm and return the result.
-    public static Boolean VerifyXml(XmlDocument Doc, RSA Key)
+    public static Boolean VerifyXml(XmlDocument xmlDoc, RSA Key)
```

Comment:
the only weird thing here is that now one param uses camelCase and the other Pascal

Reply:
I've changed the casing of Key in both source code files as well. Thanks for noticing Doc here.